src/main/java/Task.java

```diff
@@ -0,0 +1,24 @@
+public class Task {
+    private boolean completed = false;
+    private final String title;
+
```

Comment:
perhaps you may want to explore using Java enums to keep track of task types across the 3 classes by creating a TaskType enum.

For instance:
```java
public enum TaskType {
    TODO("T"),
    DEADLINE("D"),
    EVENT("E");
    private final String icon;

    TaskType(String icon) {
        this.icon = icon;
    }

    public String getIcon() {
        return icon;
    }
}
```

```suggestion
    protected TaskType type;
```